frontend/libs/health-department/feature-case-detail/src/lib/case-detail/case-detail.component.ts

```diff
@@ -254,18 +116,14 @@ export class CaseDetailComponent implements OnDestroy {
   closeCase(halResponse: HalResponse) {
     this.subs.sink = this.apiService
       .deleteApiCall<any>(halResponse, 'conclude')
-      .pipe(switchMap(() => this.healthDepartmentService.getCase(this.caseId)))
+      .pipe(switchMap(() => this.entityService.getByKey(this.caseId)))
```

Comment:
der delete call gibt bereits den aktualisierten case zurück, ein erneuter getByKey call is imho nicht notwendig

Reply:
Die Beiden Calls geben verschiedene Objekte zurueck. 
status des DELETE ist zB "CONCLUDED" und status des GET ist "abgeschlossen"